src/pages/kimdongki/Main/KimMain.js

```diff
@@ -1,7 +1,305 @@
+                className="feed_album"
+                alt="feed_album"
+                src="./images/kimdongki/song.JPG"
+              />
+              <div className="feed_icons">
+                <img
+                  className="heart_icon"
+                  alt="heart_icon"
+                  src="./images/kimdongki/heart.png"
+                />
+                <img
+                  className="comment_icon"
+                  alt="comment_icon"
+                  src="./images/kimdongki/comment.png"
+                />
+                <img
+                  className="message_icon"
+                  alt="message_icon"
+                  src="./images/kimdongki/message.png"
+                />
+                <img
+                  className="bookmark_icon"
+                  alt="bookmark_icon"
+                  src="./images/kimdongki/bookmark.png"
+                />
+              </div>
+              <div className="likes">
+                <div className="user">
+                  <div className="like_image">
+                    <img
+                      alt="love_image"
+                      src="./images/kimdongki/johnLegend.png"
+                    />
+                  </div>
+                  <span className="like_people">johnLegend</span>
+                </div>
+                <p>
+                  님 <span className="people">외 999,999명</span>이 좋아합니다
+                </p>
+              </div>
+              <div className="feed_text">
+                <p className="feedText">
+                  <span className="myNickName">d___k222</span> 김동기 '사랑의
+                  차이'
+                </p>
+                <div className="newComment">
+                  <span className="userNickName">johnLegend</span>
+                  <span className="Comment">
+                    what a wonderful voice...🤩 살람해요 킴동키😍
+                  </span>
+                  <img
+                    className="comment_icon"
+                    alt="commentHeart_icon"
+                    src="./images/kimdongki/heart.png"
+                  />
+                </div>
+              </div>
+              <ul>
+                {commentArray.map((comment, index) => (
+                  <CommentList key={index} comment={comment} />
+                ))}
+              </ul>
+              <p className="time">1분 전</p>
+              <div className="user_comment">
+                <input
+                  className="userCommet"
+                  onChange={handleCommentInput}
+                  value={comment}
+                  onKeyPress={pressEnter}
+                  type="text"
+                  id="commentAdd"
+                  placeholder="댓글 달기..."
+                />
+                <button
+                  type="button"
+                  onClick={onSubmit}
+                  className="post_btn"
+                  id="submit"
+                >
+                  게시
+                </button>
+              </div>
```

Comment:
- form 태그도 한번 사용해보시면 좋을 것 같습니다!
- id 사용은 최대한 지양해 주세요. react 특성상 재사용 가능하면 id는 더이상 고유해지지 않습니다. 

```suggestion
              <form className="user_comment" onSubmit={onSubmit}>
                <input
                  className="userCommet"
                  onChange={handleCommentInput}
                  value={comment}
                  type="text"
                  placeholder="댓글 달기..."
                />
                <button
                  type="submit"
                  className="post_btn"
                >
                  게시
                </button>
              </div>
```